requirements.txt

```diff
@@ -1,11 +1,11 @@
-Django==2.0
+Django~=2.2
 celery==4.1.1
 django-bootstrap-form==3.2.1
 oauthlib==2.0.1
 requests==2.20.0
 git+https://github.com/Aalto-LeTech/django-lti-login.git@master#egg=django-lti-login
 cachetools==2.0.1
-git+https://github.com/raphendyr/raphendyr-django-essentials.git@1.2.1
+raphendyr-django-essentials~=1.3.0
```

Comment:
I'm writing to the style guide, that there should be space here like with any operators in Python, but it's fine for now (no style guide to use as argument to do differently).